integration/token/fungible/sdk/endorser/sdk.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package endorser
+
+import (
+	"context"
+	errors2 "errors"
+	"reflect"
+
+	"github.com/hyperledger-labs/fabric-smart-client/pkg/node"
+	dig2 "github.com/hyperledger-labs/fabric-smart-client/platform/common/sdk/dig"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/fabric/services/endorser"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/driver"
+	"github.com/hyperledger-labs/fabric-token-sdk/integration/token/fungible/views"
+	"github.com/pkg/errors"
+	"go.uber.org/dig"
+)
+
+type SDK struct {
+	dig2.SDK
+	registry node.Registry
+}
+
+func NewSDK(registry node.Registry) *SDK {
+	return &SDK{registry: registry}
+}
+
+func NewFrom(sdk dig2.SDK) *SDK {
+	return &SDK{SDK: sdk}
+}
+
+func (p *SDK) Install() error {
+	// get dig from registry, this was installed by the FTS's sdk
+	var digContainer *dig.Container
+	if p.SDK != nil {
+		if err := p.SDK.Install(); err != nil {
+			return err
+		}
+		digContainer = p.SDK.Container()
+	} else {
+		// get it from the registry
+		if p.registry == nil {
+			return errors.New("registry not set")
+		}
+		wrapper, err := p.registry.GetService(reflect.TypeOf((*dig.Container)(nil)))
```

Comment:
Registering the container as a service doesn't look right. It is available from `p.Container()`. Let's not introduce multiple ways of accessing it, because it becomes too complex to manage.